src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterRepository.java

```diff
@@ -120,6 +122,22 @@ public int updateIsFailed(UUID id) {
         );
     }
 
+    /**
+     * Retrieve a list of letters which are not failed but still not printed.
+     *
+     * @return a list of unprinted letters which were sent before yesterdays 5pm deadline
+     */
+    public List<NotPrintedLetter> getStaleLetters() {
+        return jdbcTemplate.query(
+            "SELECT id, message_id, service, type, created_at, sent_to_print_at "
+                + "FROM letters "
+                + "WHERE is_failed = false"
+                + "  AND printed_at IS NULL"
+                + "  AND sent_to_print_at IS NOT NULL"
+                + "  AND sent_to_print_at < (CURRENT_DATE - integer '1' + time '17:00')",
```

Comment:
Do you think it will be good to put `15:00` in `application.yml` rather than hardcoding in code say tomorrow timing changes?

Reply:
An idea, yes. Could strip `5pm` from return description too. But then why not configure the whole thing and make it more complicated. on the other hand, it is related (I guess) with ftp availability which is defined in consumer.

I would prefer to leave this for a feature extension

Reply:
perhaps we could just specify a number of hours that passed since the letter was sent to print